client/my-sites/plans/jetpack-plans/storage-tier-upgrade/style.scss

```
@@ -0,0 +1,18 @@
+.storage-tier-upgrade {
+	display: grid;
+	grid-template-columns: repeat( auto-fit, minmax( 300px, 1fr ) );
+	gap: 24px 16px;
+
+	margin-top: 48px;
+	margin-bottom: 64px;
+
+	.jetpack-product-card__product-name {
+		font-size: 3.375rem;
+		margin-bottom: 9px;
+	}
+
+	.jetpack-product-card__product-subheader {
+		font-size: 2rem;
+		margin-top: 9px;
+	}
```

* According to `1a5ZL6xHHfOXdpXYQdiZuT-fi-774%3A0`, this font-size should be 24px, or `$font-title-medium`.
* Letter spacing is decreased here as well, but since the font size is smaller, 2px would compress the text too much -- 1px should work fine.
* Text is also center-aligned for this part of the header.

```suggestion
	.jetpack-product-card__product-subheader {
		font-size: $font-title-medium;
		letter-spacing: -1px;
		text-align: center;
		margin-top: 9px;
	}
```